react-frontend/src/utils/helper.js

```diff
@@ -57,3 +57,15 @@ export const calcPer = (uptime, downtime) => {
 //   });
 //   return arrayStr.join("<br/>");
 // }
+
+export const setUerId = (id) => {
+  localStorage.setItem('stat_user_id',id);
+}
+
+export const getUserId = () => {
+  return localStorage.getItem('stat_user_id');
+}
+
+export const generateUUID = (length) => {
+  return Array.from(Array(length), () => Math.floor(Math.random() * 36).toString(36)).join('')
```

Comment:
I guess this would suffice `Math.floor(Math.random() * 36).toString(36))`
No need for `Array.from(...)`

Reply:
No it is required, as `Math.floor(Math.random() * 36).toString(36)` generates random number with base 36 and returns only single character, with Array.from() in place it takes length as parameter and loops over every element to generate id of length passed